config/development.yaml

```diff
@@ -1,5 +1,8 @@
 log_level: DEBUG
 
+feature_flags:
+  use_new_shard_key_mapping_format: false
+
```

Comment:
This idea immediately paid off, enabling the new shard key mapping logic turns out to be problematic with recent changes. A few consensus tests fail and it's not immediately clear to me what the problem is.

I'll disable it for now, and resolve the problems in a separate PR.

We can merge this one :+1: